src/components/PupilViews/PupilExperience/PupilExperience.view.tsx

```diff
@@ -0,0 +1,122 @@
+import {
```

Comment:
It would have been a good thing to lift this out of the page regardless. It was getting messy in there.